Extension/package.json

```diff
@@ -3078,6 +3078,17 @@
                         "maximum": 65536,
                         "scope": "machine"
                     },
+                    "C_Cpp.maxSymbolSearchResults": {
+                        "type": [
+                            "integer",
+                            "null"
+                        ],
+                        "markdownDescription": "%c_cpp.configuration.maxSymbolSearchResults.markdownDescription%",
+                        "default": 5000,
+                        "minimum": 500,
+                        "maximum": 10000,
```

Comment:
Can the 10k max be increased to 20000 or is that something we don't want for some reason?

Reply:
I can't see a reason why 20000 results is something the user would **actually** want. 

What happens if you get 20000 results? You're not going to find what you're looking for. 

That's 800 pages of results in the UI.

Reply:
Yeah, I'm thinking about the initial query being 20k but results in a lot less. The user probably can't tell 10k vs. 20k results.

Reply:
I don't think we should allow 20,000

Reply:
sure, I'm fine with that